src/commands/createNewProject/createNewProject.ts

```diff
@@ -106,3 +110,20 @@ export function getProjectCreator(language: string, functionAppPath: string, act
             return new ScriptProjectCreatorBase(functionAppPath, actionContext.properties);
     }
 }
+
+async function addToFilesExclude(filesToExclude: string | string[], fsPath: string): Promise<void> {
+    const exclude: string = 'exclude';
+    const projectConfiguration: WorkspaceConfiguration = workspace.getConfiguration('files', Uri.file(fsPath));
```

Comment:
If you're editing settings, it should be done in the `writeVSCodeSettings` function in `initProjectForVSCode`:
https://github.com/Microsoft/vscode-azurefunctions/blob/master/src/commands/createNewProject/initProjectForVSCode.ts#L86

If you're not familiar with 'init project', take a look at the "Create new project from func cli" test case in our OneNote.